project01.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Elson Otake</title>
+</head>
+<body>
+  <div class="project ellipse_03">
+    <h1 class="work theme43">MY RECENT WORK</h1>
+    <hr class="line_3">
+    <section class="proj_sect">
+      <img class="proj_img" src="images/projects-79906734.jpeg" alt="Image of project collected from iStock"/>
+      <h4 class="proj_title themefff">Multi-Post Stories</h4>
+      <p class="proj_txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    </section>
+    <ul class="proj_group">
+      <li class="proj_lang themefff"><a href="#">Css</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">html</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Javascript</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Ruby</a></li>
+    </ul>
+  </div>
+  <button type="button" class="proj_button theme44">See project</button>
```

Comment:
- [ ] As a core requirement, everything here should be in a `<section>` tag. Please surround all the content in a `<section>` tag 👍